### PR TITLE
Transform prerendered frontpage's links to relative links

### PR DESCRIPTION
**Pull request by kaol**

### CODE REVIEW

File: packages/components/src/HtmlRenderer.js [hunk 1]
```diff
@@ -45,6 +33,7 @@ exports.renderHtmlInputWithHooksImpl = function (htmlInput, hooks) {
     return htmlToReactParser.parseWithInstructions(
         htmlInput,
         isValidNode,
-        uncurriedHooks
+	uncurriedHooks,
+	uncurriedPreprocessHooks
```
Comment (ae-mo) [inline packages/components/src/HtmlRenderer.js:37]: `uncurriedHooks` comes from a previous version where I was uncurrying purescript functions manually, now it's not the case anymore. Maybe we should change these names.

File: packages/components/src/HtmlRenderer/Models.purs [hunk 2]
```diff
@@ -108,31 +109,35 @@ modifyingHook = toHookRep <<< ModifyingHook
 instance modifyingHookToGenericHook :: ToGenericHook ModifyingHook where
   toGenericHook (ModifyingHook { shouldProcessNode, processNode }) =
     { replaceChildren: false
-    , shouldProcessNode
-    , processNode: Just processNode
-    , processNodeWithReplacement: Nothing
+    , shouldPreprocessNode: Just shouldProcessNode
+    , shouldProcessNode: Just shouldProcessNode
```
Comment (ae-mo) [inline packages/components/src/HtmlRenderer/Models.purs:113]: Can this be `Nothing`?

Comment (kaol) [reply packages/components/src/HtmlRenderer/Models.purs:113]: Yes indeed. A left over from my testing.

File: packages/components/src/HtmlRenderer.js [hunk 3]
```diff
@@ -19,23 +19,11 @@ exports.renderHtmlInputWithHooksImpl = function (htmlInput, hooks) {
...
-          shouldProcessNode: h.shouldProcessNode,
-          processNode,
-        }
-    });
+    const processHooks = hooks.filter(h => h.shouldProcessNode);
+    const preprocessHooks = hooks.filter(h => h.shouldPreprocessNode);
 
     // A catch-all hook
-    uncurriedHooks.push({
+    processHookds.push({
```
Comment (ae-mo) [inline packages/components/src/HtmlRenderer.js:26]: Typo

**Review by ae-mo — APPROVED** [review]

**Pull request merged**